From 957f894a122aa3a8dd8954aa35a1042726a49363 Mon Sep 17 00:00:00 2001
From: Your Name <you@example.com>
Date: Mon, 11 May 2026 08:59:59 +0000
Subject: [PATCH 1/3] Add RTW89_FLAG_PROBE_DONE flag

Based on "wifi: rtw89: fw: read firmware secure information from efuse"
---
 src/core.h | 1 +
 src/pci.c  | 2 ++
 2 files changed, 3 insertions(+)

diff --git a/src/core.h b/src/core.h
index a85bb77..109429b 100644
--- a/src/core.h
+++ b/src/core.h
@@ -3858,6 +3858,7 @@ enum rtw89_flags {
 	RTW89_FLAG_POWERON,
 	RTW89_FLAG_FW_RDY,
 	RTW89_FLAG_RUNNING,
+	RTW89_FLAG_PROBE_DONE,
 	RTW89_FLAG_BFEE_MON,
 	RTW89_FLAG_BFEE_EN,
 	RTW89_FLAG_BFEE_TIMER_KEEP,
diff --git a/src/pci.c b/src/pci.c
index 33b2543..1f127a5 100644
--- a/src/pci.c
+++ b/src/pci.c
@@ -3978,6 +3978,8 @@ int rtw89_pci_probe(struct pci_dev *pdev, const struct pci_device_id *id)
 		goto err_free_irq;
 	}
 
+	set_bit(RTW89_FLAG_PROBE_DONE, rtwdev->flags);
+
 	return 0;
 
 err_free_irq:
-- 
2.39.5

